scipy/datasets/_fetchers.py

```diff
@@ -71,7 +71,7 @@ def ascent():
     fname = fetch_data("ascent.dat")
     # Now we just need to load it with our standard Python tools.
     with open(fname, 'rb') as f:
-        ascent = array(pickle.load(f))
+        ascent = array(pickle.load(f), dtype='uint8')
```

Comment:
```suggestion
        ascent = array(pickle.load(f))
```

Not required anymore, see https://github.com/scipy/dataset-ascent/pull/3

Reply:
@AnirudhDagar Which one  of the two solutions would you like me to take a look at? "Two solutions: Either change w to float32 or change image to float64 in the arguments for signal.sepfir2d(image, w, w)"

Reply:
Let's cast the `image` to `np.float64` when passing to `sepfir2d`. Should fix it!